templates/base/packages/nextjs/app/layout.tsx.template.mjs

```diff
@@ -9,10 +9,7 @@ import { ThemeProvider } from "~~/components/ThemeProvider";
 import "~~/styles/globals.css";
 import { getMetadata } from "~~/utils/scaffold-eth/getMetadata";
 
-export const metadata = getMetadata({
-  title: "Scaffold-ETH 2 App",
-  description: "Built with 🏗 Scaffold-ETH 2",
-});
+export const metadata = getMetadata(${metadata});
```

Comment:
this could be done with `{...${JSON.stringify(metadata)}}` again or `{title: ${metadata.title}, description: ${metadata.description}}` but not sure about both for now. Probably we can do it later when decide

Reply:
![image](https://github.com/user-attachments/assets/0c9076f8-5486-4b9e-915f-1a09b78fdfbb)

As we talked about, in some of these cases (like this one), having an object seems like the obvious choice, right?

But yeah, allowing both might be confusing. And also you mentioned that there were some limitations. It'd be nice to have the "rules" for template literals VS  objects/arrays.

Anyway, just commenting here (it seems that we will be closing #116)... so ignore my comment for this PR haha.

Reply:
Updated to object [5955054](https://github.com/scaffold-eth/create-eth/pull/131/commits/59550544d029e4b2da2bf30f681fb98f0eb6c273) .

as a note, we can update menuLinks to objects too, but it's better to do it when #132 will be approved/merged and in separate pr